bciers/apps/reporting/src/app/components/reportInformation/nonAttributableEmissions/NonAttributatbleEmissions.tsx

```diff
@@ -0,0 +1,29 @@
+import { Suspense } from "react";
+import Loading from "@bciers/components/loading/SkeletonForm";
+import NonAttributableEmissionsForm from "@reporting/src/app/components/reportInformation/nonAttributableEmissions/NonAttributableEmissionsForm";
+import { UUID } from "crypto";
+import { getAllGasTypes } from "@reporting/src/app/utils/getAllGasTypes";
+import { getAllEmissionCategories } from "@reporting/src/app/utils/getAllEmissionCategories";
+
+interface NonAttributableEmissionsProps {
+  versionId: number;
+  facilityId: UUID;
+}
+
+export default async function NonAttributableEmissions({
```

Comment:
I think this component is missing some functionality where it should load any existing non-attributable records. When I save data & then navigate back to the /non-attributable page, it doesn't load the data that has been saved. Likely needs an additional api endpoint/service to pull existing data & parse it into the shape of the formData.